apps/launchpad/app/routes/_app+/quests+/questions+/$epochId+/$questionId.tsx

```diff
@@ -86,50 +69,22 @@ import { formatUnits } from 'viem'
 
 export async function loader({ request, params }: LoaderFunctionArgs) {
   const queryClient = new QueryClient()
-  const url = new URL(request.url)
 
-  // Get pagination params from URL with defaults
-  const pageSize = parseInt(url.searchParams.get('limit') ?? '20', 10)
-  const pageIndex = Math.max(
-    0,
-    parseInt(url.searchParams.get('page') ?? '1', 10) - 1,
-  )
+  // Start parallel fetches for independent data
```

Comment:
Awesome -- note to ourselves that we have to use this Promise.all pattern to avoid serial fetches when in the loader

React Query on the client has more robust options for parallel fetching (and does so by default)